### ISSUE TITLE
Dynamically deploy vmware machine for testing

### ISSUE THREAD

**Issue by chambridge** [quipucords/rho#72]
Create ansible playbook/roles for deploying different VMware Snapshots of RHEL versions and Windows.
Extend deployment by adding JBoss deployment role.

**Comment by chambridge** [comment]
@elyezer I started looking at this some on Friday. Been running into a few issues obtaining an IP for the created snapshot. I'm going to reach out to Chris Roberts to see if he has any ideas, but worst case I was thinking we could essentially have a set of VMs that we made sure were powered on via an Ansible script before testing and then power them off after testing. Since the scan doesn't alter anything on the machine there is no big difference between just turning on a VM vs. creating a clone. Let me know your thoughts / concerns.
